lib/rucio/core/replica.py

```diff
@@ -72,6 +77,13 @@
                                    DEFAULT_SCHEMA_NAME, BASE)
 from rucio.rse import rsemanager as rsemgr
 
+logging.basicConfig(stream=sys.stdout,
+                    level=getattr(logging,
+                                  config_get('common', 'loglevel',
+                                             raise_exception=False,
+                                             default='ERROR').upper()),
+                    format='%(asctime)s\t%(process)d\t%(levelname)s\t%(message)s')
+
```

Comment:
This must be removed. `setup_logging` from `rucio.common.logging` is run at entrypoints.

Reply:
agree